code/modules/mob/living/simple_animal/abnormality/aleph/staining_rose.dm

```diff
@@ -0,0 +1,147 @@
+#define STATUS_EFFECT_WILTING /datum/status_effect/wilting
+//Coded by me, Kirie Saito!
+/mob/living/simple_animal/hostile/abnormality/staining_rose
+	name = "Staining Rose"
+	desc = "A tiny, wilting rose."
+	icon = 'ModularTegustation/Teguicons/64x64.dmi'
+	icon_state = "rose"
+	maxHealth = 10
+	health = 10			//It's a rose lol
+	threat_level = ALEPH_LEVEL
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = 0,
+		ABNORMALITY_WORK_INSIGHT = list(0, 0, 0, 30, 40),
+		ABNORMALITY_WORK_ATTACHMENT = 0,
+		ABNORMALITY_WORK_REPRESSION = list(0, 0, 0, 45, 50)
+			)
+	start_qliphoth = 3
+	work_damage_amount = 16
+	work_damage_type = PALE_DAMAGE
+	pixel_x = -16
+	base_pixel_x = -16
+	pixel_y = -8
+	base_pixel_y = -8
+
+	ego_list = list(
+//		/datum/ego_datum/weapon/blooming,
+//		/datum/ego_datum/armor/blooming
+		)
+
+	var/working = FALSE
+	var/chosen
+	var/meltdown_cooldown_time = 300 SECONDS
+	var/meltdown_cooldown
+
+
+/mob/living/simple_animal/hostile/abnormality/staining_rose/work_complete(mob/living/carbon/human/user, work_type, pe)
+	..()
+	if (working==FALSE)
+		working = TRUE
+		chosen = user
+		user.visible_message("<span class='warning'>You are now the rose's chosen.</span>")
+		icon_state = "rose_activated"
+
+	if (user != chosen)		//Your body starts to wilt.
+		user.visible_message("<span class='warning'>The rose has already chosen another, [chosen]!</span>")
+		user.physiology.red_mod *= 1.2
+		user.physiology.white_mod *= 1.2
+		user.physiology.black_mod *= 1.2
+		user.physiology.pale_mod *= 1.2
+		datum_reference.qliphoth_change(-1)
+		pissed()
+
+	else
+		datum_reference.qliphoth_change(1)
+		user.visible_message("<span class='warning'>The Rose is satisfied.</span>")
+
+	if(get_attribute_level(user, JUSTICE_ATTRIBUTE) < 100)
+		user.visible_message("<span class='warning'>The Rose is sapping your strength</span>")
+		user.physiology.red_mod *= 1.1
+		user.physiology.white_mod *= 1.1
+		user.physiology.black_mod *= 1.1
+		user.physiology.pale_mod *= 1.1
+		pissed()
+
+/mob/living/simple_animal/hostile/abnormality/staining_rose/Life()
+	. = ..()
+	if(meltdown_cooldown < world.time)
```

Comment:
Holy shit, WN mechanics.
Add some warning, at least a sound.

Reply:
Nah, Lobotomy Corp fans want this, trust me.